### PR TITLE
Tabletop Exercise Blog Post

### PR DESCRIPTION
**Pull request by dggreenbaum**

### CODE REVIEW

File: _posts/2023-09-29-tabletop-incident-response-exercises-for-small-teams/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+- Roll for more information.
+- Ask questions of the facilitator (me).
+- Escalate to get another Engineer on the call.
+    - The new engineer can [assist with die rolls](https://www.dndbeyond.com/sources/basic-rules/using-ability-scores#AdvantageandDisadvantage) and re-roll prior rolls to get more information
+- Take an action. This is anything they can think of and explain how to do.
+
+This ended being a rather thorough evaluation of a readiness to respond:
+
+- Several types of incident were explored, including Anomalous Load, a Security Breach, and Data Loss.
```
Comment (jkemp101) [inline _posts/2023-09-29-tabletop-incident-response-exercises-for-small-teams/index.md:48]: incidents

File: _posts/2023-09-29-tabletop-incident-response-exercises-for-small-teams/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- Dynamism. How much time was spent acting vs spinning wheels?
+    - 1 - No action was taken within the time frame of the exercise
+    - 2 - Action was taken after prompting from those running the exercise
+    - 3 - Discordant or incorrect action was taken without prompting
+    - 4 - Correct action was taken without prompting
+    - 5 - Harmonious and correct action was taken without prompting
+- Communication. Were the proper tools used the proper way?
+    - 1 - Scenario was disregarded. Most communication occurred outside the rules of the exercise
+    - 2 - Scenario partially respected. Minimal communication occurred outside of the rules
+    - 3 - Scenario fully respected. No communication occurred outside of the rules.
```
Comment (jkemp101) [inline _posts/2023-09-29-tabletop-incident-response-exercises-for-small-teams/index.md:70]: There are no periods in previous list items

**Review by jkemp101 — APPROVED** [review]
Two small comments

File: _posts/2023-09-29-tabletop-incident-response-exercises-for-small-teams/index.md (hunk 1 above)
Comment (dggreenbaum) [reply _posts/2023-09-29-tabletop-incident-response-exercises-for-small-teams/index.md:48]: I disagree, but don't have the linguistic term to provide a source. EG:

> Several types of duck were available.
> You may choose any type of car you want.

Comment (jkemp101) [reply _posts/2023-09-29-tabletop-incident-response-exercises-for-small-teams/index.md:48]: True, you are probably correct. Rewording to Several incident types makes it sound right.

**Comment by philfreo** [comment]
Cool post!

**Pull request merged**